src/interviewer.t

```diff
@@ -217,17 +235,70 @@ warner_interviewer : Person 'interviewer' 'Harried Interviewer' @interview_room
 ;
 
 ++ SpecialTopic 'anime club' ['anime', 'anime club', 'japanese', 'japanese club', 'weeaboo', 'weeb']
-    ""
+    ""//"<q>私はアニメが大好きです~~~!</q>"
```

Comment:
and you're certain that whatever you're using as a terminal supports Unicode? TADS3 uses Unicode under the hood, but if it's not showing up, either your interpreter or terminal doesn't support it / deal with it properly.